website/sidebars.js

```diff
@@ -255,6 +255,7 @@ module.exports = {
                                 'reference/sdks/react',
                                 'reference/sdks/svelte',
                                 'reference/sdks/vue',
+                                'reference/sdks/flutter-proxy',
```

Comment:
Again, please make sure that this list remains alphabetical: move the flutter docs to before JavaScript.